eng-wasm/src/crypto_wasm.rs

```diff
@@ -17,41 +16,15 @@ pub fn generate_key() -> SymmetricKey {
 }
 
 pub fn encrypt(message: &[u8], key: &SymmetricKey) -> Vec<u8> {
-    // TODO: Is this really needed? Dynamically sized buffers don't seem to work.
-    // TODO: Is it possible to estimate the encrypted payload size based on the plaintext message?
-    let length = PAYLOAD_SIZE;
-    let mut payload = Vec::with_capacity(length);
-    for _ in 0..length {
-        payload.push(0);
-    }
+    let length = message.len() + IV_SIZE + TAG_SIZE;
```

Comment:
Shouldn't you use `extra_size_for_cypher` instead of `IV_SIZE + TAG_SIZE` here?
Also, this is not the calculation that itzik demostrated in #230. Can you please explain this new calculation?